safe/report/processors/default.py

```diff
@@ -592,3 +605,113 @@ def qt_svg_to_png_renderer(impact_report, component):
 
     component.output = output_path
     return component.output
+
+
+def atlas_renderer(composition, coverage_layer, output_path, file_format):
+    """Extract composition using atlas generation.
+
+    :param composition: QGIS Composition object used for producing the report
+    :type composition: qgis.core.QgsComposition
+
+    :param coverage_layer: Coverage Layer used for atlas map
+    :type coverage_layer: QgsMapLayer
+
+    :param output_path: the output path of the product
+    :type output_path: str
+
+    :param file_format: file format of map output, PDF or PNG
+    :type file_format: 'pdf', 'png'
+
+    :return: generated output path(s)
```

Comment:
We usually put capital letter and dot for `:return:` and `:param:` as @ismailsunni said ;-)